db/migrations/0001_add_minting_transaction_table.sql

```diff
@@ -0,0 +1,7 @@
+-- Migration number: 0001 	 2025-05-06T06:27:42.779Z
+CREATE TABLE minting_transactions (
+    txid TEXT PRIMARY KEY,
+    status TEXT NOT NULL CHECK(status IN ('pending', 'confirmed', 'failed'))
```

Comment:
```suggestion
    status INTEGER NOT NULL CHECK(status IN (0,1,2,3))
```

Reply:
add comment about enum values and create constants in the Remix TS code

Reply:
What does 0,1,2 and 3 means?

Reply:
status
0 = none (should not be used), so actually we should exclude it from check
1 = failed
2 = pending
3 = finalized

Reply:
Done